### ISSUE TITLE
[Feature Request] Underlying OS Update Status

### ISSUE THREAD

**Issue by iAlchemy** [IceWhaleTech/CasaOS#1094]
### Description

It would be nice if CasaOS could indicate if updates for the underlying OS were available and provide a means to install them.

Yes ... you can do this manually from Terminal, but it would be nice to have CasaOS check periodically and then when requested to update, open a terminal window and post the appropriate command.

### Additional Information

	 - OS: linux
	 - CasaOS Version: 0.4.3
	 - Disk Total: 62179 
	 - Disk Used: 16142 
	 - System Info: [Ubuntu 22.04.2 LTS   \l Bit:64 Mem:3923M Core:4 Linux casaos 5.15.0-71-generic #78-Ubuntu SMP Tue Apr 18 09:00:29 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux]
	 - Remote Version: 0.4.3
	 - Browser: Chrome 
	 - Version: 113.0.0.0

**Comment by tigerinus** [comment]
Interesting idea...

I think a better to make this possible, is to allow custom toast popups in CasaOS dashboard, and it should allow custom buttons like `Yes` and `No`.

Once this mechanism is in place, then 3rd party developers can contribute a *OS Updater* (docker or non-docker - doesn't matter) and interact with users.
